docs/usage/getting-started/private-packages.md

```diff
@@ -457,7 +457,7 @@ name = "pypi"
 
 ### poetry
 
-For every Poetry source, a `hostRules` search is done and then any found credentials are added to env like `POETRY_HTTP_BASIC_X_USERNAME` and `POETRY_HTTP_BASIC_X_PASSWORD`, where `X` represents the normalized name of the source in `pyproject.toml`.
+For every Poetry source, a `hostRules` search is done and then any found credentials are added to env like `POETRY_HTTP_BASIC_X_USERNAME` and `POETRY_HTTP_BASIC_X_PASSWORD`, or `POETRY_PYPI_TOKEN_X`, where `X` represents the normalized name of the source in `pyproject.toml`.
```

Comment:
```suggestion
For every Poetry source, a `hostRules` search is done and then any found credentials are added to env like `POETRY_HTTP_BASIC_X_USERNAME` and `POETRY_HTTP_BASIC_X_PASSWORD`, or `POETRY_HTTP_TOKEN_X`, where `X` represents the normalized name of the source in `pyproject.toml`.
```
please align naming to existing environment variables

Reply:
Hello! 
This is not a choice, this naming is required by poetry it self: POETRY_PYPI_X_TOKEN
cf https://python-poetry.org/docs/repositories/#configuring-credentials:~:text=Alternatively%2C%20you%20can%20use%20environment%20variables%20to%20provide%20the%20credentials%3A